WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsUnknownView.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+import SwiftUI
+
+final class CardReaderSettingsUnknownViewController: UIHostingController<CardReaderSettingsUnknownView>, CardReaderSettingsViewModelPresenter {
+    func configure(viewModel: CardReaderSettingsPresentedViewModel) {
+        guard let viewModel = viewModel as? CardReaderSettingsUnknownViewModel else {
+            DDLogError("Unexpectedly unable to downcast to CardReaderSettingsUnknownViewModel")
+            return
+        }
+        rootView.viewModel = viewModel
+    }
+
+    init(viewModel: CardReaderSettingsUnknownViewModel) {
+        super.init(rootView: CardReaderSettingsUnknownView(viewModel: viewModel))
+    }
+
+    @objc required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+struct CardReaderSettingsUnknownView: View {
+    @State var viewModel: CardReaderSettingsUnknownViewModel
+    @State var presentedURL: URL? = nil
+
+    var body: some View {
+        ScrollView {
+            VStack(spacing: 20) {
+                Text(Localization.connectYourCardReaderTitle)
+                    .font(.headline)
+                    .frame(height: 90)
+
+                Image("card-reader-connect")
+
+                NumberedListItem(number: Localization.hintOneTitle, content: Localization.hintOne)
+                NumberedListItem(number: Localization.hintTwoTitle, content: Localization.hintTwo)
+                NumberedListItem(number: Localization.hintThreeTitle, content: Localization.hintThree)
+
+                Spacer()
+
+                Button(action: {}) {
+                    Text(Localization.connectButton)
+                        .frame(maxWidth: .infinity)
+                }
+                .buttonStyle(PrimaryButtonStyle())
+
+                HStack(alignment: .center, spacing: 20) {
+                    Image(uiImage: .infoOutlineImage)
+                        .accentColor(Color(.lightGray))
+                        .frame(width: 20, height: 20)
+                    AttributedText(Localization.learnMore)
```

Comment:
See comment below about support coming in iOS15. Maybe we could negotiate with design to avoid having to do this (i.e. by making the entire text a link)